app/script_variant.py

```diff
@@ -145,6 +145,53 @@ def romanize_word_by_unidecode(word):
     romanized_word = unidecode(word)
     return [romanized_word.strip()]
 
+def is_chinese_person_name(word):
+    """
+    This method checks if a unicode string is a chinese person name by figuring out
+    if it contains valid a chinese surname and a given name.
+    
+    Return:
+        Boolean
+    """
+    surname, lastname  = split_chinese_name(word)
+    return surname is not None and lastname is not None
+
+def split_chinese_name(word):
+    """
+    This method tries to split a chinese name into two parts: surname and lastname
+    Args:
+        word: a chinese name string
+    Returns:
+        surname, lastname if it is a valid chinese name
+        else None, None
+    """
+    word = word.replace(" ", "")
+    if len(word) != len(re.findall(ur"[\u4e00-\u9fa5]", word)):
```

Comment:
Use code range \u3400-\u9fff instead, which is used here:
https://github.com/google/personfinder/blob/bcec681718ae454ced01d00fe839026b18ab80ad/app/text_query.py#L44
It looks your range misses some characters.

Even \u3400-\u9fff doesn't look like a comprehensive range, but we can take a look later.

Also, can you simplify this to e.g.;

```python
if not re.search(ur'^[\u3400-\u9fff]$', word):
```